restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/entities/shiro/Realm.java

```diff
@@ -96,7 +96,7 @@ protected AuthenticationInfo doGetAuthenticationInfo(final AuthenticationToken t
             }
 
             if (account != null) {
-                authToken = account.getAuthToken();
+                authToken = account.getAuthToken().get(0).getAuthToken();//Not sure if this is the right approach
```

Comment:
@arslan70 why use the 1st auth token only? This doesn't seem right. Even though as far as I remember this is not used anyore.